projects/container_bench/testing/exec_time.py

```
@@ -69,33 +276,47 @@
-        stderr=subprocess.PIPE,
-        text=True,
-        bufsize=1,
-        universal_newlines=True
-    ) as process:
-        stdout, stderr = process.communicate()
-        process.wait()
-    end_time = time.perf_counter()
-    stop_event.set()
-    monitor_thread.join()
+    try:
+        with subprocess.Popen(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)